image/templates/helm/shared/templates/99-scanner-v4-openshift-monitoring.yaml

```diff
@@ -0,0 +1,57 @@
+{{- include "srox.init" . -}}
+{{- if ._rox.monitoring.openshift.enabled -}}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: "scanner-v4-monitor-{{ .Release.Namespace }}"
+  namespace: openshift-monitoring
+  labels:
+    {{- include "srox.labels" (list . "servicemonitor" (print "scanner-v4-monitor-" .Release.Namespace)) | nindent 4 }}
+  annotations:
+    {{- include "srox.annotations" (list . "servicemonitor" (print "scanner-v4-monitor-" .Release.Namespace)) | nindent 4 }}
+spec:
+  endpoints:
+  - interval: 10s
+    path: metrics
+    port: indexer-monitoring-tls
+    scheme: https
+    tlsConfig:
+      caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
+      certFile: /etc/prometheus/secrets/metrics-client-certs/tls.crt
+      keyFile: /etc/prometheus/secrets/metrics-client-certs/tls.key
+      serverName: "scanner-v4-indexer.{{ .Release.Namespace }}.svc"
+  - interval: 10s
+    path: metrics
+    port: matcher-monitoring-tls
+    scheme: https
+    tlsConfig:
+      caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
+      certFile: /etc/prometheus/secrets/metrics-client-certs/tls.crt
+      keyFile: /etc/prometheus/secrets/metrics-client-certs/tls.key
+      serverName: "scanner-v4-matcher.{{ .Release.Namespace }}.svc"
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: scanner-v4
+  namespaceSelector:
+    matchNames:
+    - "{{ .Release.Namespace }}"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
Can you create a `ClusterRoleBinding` for this? See https://issues.redhat.com/browse/ROX-22188, we want to avoid leaking resources into other namespaces if possible.